src/single-spa-child-app-error.js

```diff
@@ -21,7 +21,13 @@ function SingleSpaChildAppError(obj, childApp) {
 		this.message = createMessage(childApp, obj.message);
 		this.stack = obj.stack;
 	} else {
-		const err = new Error(obj || 'An error has occurred within a ChildApp of single-spa');
+		let msg;
+		try {
+			msg = JSON.stringify(obj);
```

Comment:
The error `App workflow-ui!sofe is misbehaving -- [Object object]` was caused by us not properly wrapping the error message here.